pandas/tests/plotting/test_frame.py

```diff
@@ -204,6 +204,23 @@ def test_color_and_style_arguments(self):
         with pytest.raises(ValueError):
             df.plot(color=["red", "black"], style=["k-", "r--"])
 
+    def test_color_and_marker(self):
+        # GH21003 2018-05-10
```

Comment:
What's `2018-05-10`? I've not seen dates elsewhere in the codebase, I think it should be removed

Reply:
I clarified these two lines.